### DISCUSSION TITLE
请求开放sdk，或者代编译部分需要引用内核模块的包

### DISCUSSION THREAD

**Discussion by FUjr** [coolsnowwolf/lede#9510] [R68S / 电犀牛专区]
你好，感谢提供rk3568固件支持。我现需要编译部分内核模块或者有一些包要引用内核头文件，用其他sdk可能不太行。请求开发组开放sdk，或者代编译
插件：UA2F  （需要hook—ipset我猜测可能需要对应头文件）项目地址-https://github.com/Zxilly/UA2F
驱动：mt7612u

**Comment by Pb-N32** [comment]
你好同求

**Comment by disorderman** [comment]
同求
